### DISCUSSION TITLE
[Script request] Falcon Player

### DISCUSSION THREAD

**Discussion by SparkyDan555** [community-scripts/ProxmoxVE#218] [Request script]
### Application Name

Falcon Player

### Website

https://github.com/FalconChristmas/fpp

### Description

Falcon Player (formally FPP - Falcon Pi Player) is an application used for managing and controlling animated light shows like you see on [The Great Christmas Light Fight](https://abc.com/shows/the-great-christmas-light-fight). It is typically run on a Raspberry Pi but can also be run on top of a standard Debian installation on any supported hardware. There is already an [install script](https://github.com/FalconChristmas/fpp/blob/master/SD/FPP_Install.sh) to install Falcon Player on top of a stock OS install. I have proven it can be installed on Proxmox on top of Debian 12 VM using this script. Not sure if an LXC would work as it is necessary to pass through USB devices such as USB sound cards / FM transmitters. It would be great to be able to install this on Proxmox with a simple script!

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Why should this be an LXC / VM? The way I read it, it's for single computer ala esp/Raspberry Pi, Bananapi.

What is the point of building this as an LXC? Most of the devices are in the home data center. Or I'm misunderstanding it.

**Reply by SparkyDan555** [reply]
It doesn't have to run on a Pi, this is just the most usual use case and is what the software was originally designed for. The point of running this as a VM is the same as every other script here - to save resources by running multiple tools on the same hardware. With Proxmox I can run Falcon player on my home server alongside Home Assistant, ESPHome, Plex, OpenMediaVault etc.
